ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/hosts.yml

```diff
@@ -10,12 +10,13 @@ test_hosts:
     host1:
     host2:
     host3:
+    host4:
```

Comment:
dont' add host4 + host4_inline_jinja as suggested here by @ClausHolbechArista plesae rename the host4_inline_jinja to host_inline_jinja / host-inline-jinja (be consistent between the two PRs)

https://github.com/aristanetworks/avd/pull/4712#discussion_r1863547321